.github/workflows/create-pr.yaml

```
@@ -19,6 +20,51 @@ permissions:
   contents: read
 
 jobs:
+  create-tentants-config-pr:
+    if: >
+      inputs.run_for == 'tentants-config' || inputs.run_for == 'both' || inputs.run_for == ''
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout tenants-config
+      uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+      with:
+        repository: redhat-appstudio/tenants-config
+        ref: main
+        path: tenants-config
+
+    - name: Update ec-policies
+      run: ./hack/update-infra-deployments.sh ../tenants-config
+      working-directory: ec-policies
+
+    - name: Checkout infra-deployments-ci
+      uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+      with:
+        path: infra-deployments-ci
```

This should be above the previous step and checkout ec-policies (so that the `hack/update-infra-deployments.sh` is available.

```suggestion
    - name: Checkout ec-policies
      uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
      with:
        repository: enterprise-contract/ec-policies
        ref: main
        path: ec-policies
```

---

This should be fixed now.